### DISCUSSION TITLE
4.2.0 release roadmap

### DISCUSSION THREAD

**Discussion by lfcnassif** [sepinf-inc/IPED#2243] [General]
All devs,

I've just updated the current release notes for 4.2.0, we have about 90 improvements:
https://github.com/sepinf-inc/IPED/blob/master/ReleaseNotes.txt

The current planned minimum features/fixes were just updated in the 4.2.0 roadmap:
https://github.com/sepinf-inc/IPED/projects/3

Should we include something else?

**Comment by gfd2020** [comment]
I believe @wladimirleite  has improved several GUI icons.

PS: Many new features implemented!

**Comment by gfd2020** [comment]
Upgrade Apache Commons Compress to 1.25.0. It will fix issues #1961 and #2328

**Reply by lfcnassif** [reply]
Done

**Comment by aberenguel** [comment]
Upgrade to JEP 4.2.0 will fix #2253.

**Comment by aberenguel** [comment]
Pull requests #2288 and #2293 are ready to be validated and merged.

**Comment by lfcnassif** [comment]
I'm going to roll 4.2 version just with current reviewed and merged changes. I apologize all people that sent PRs pending for review, sorry for not having time to review and test all of them.
